content/security/rate-limiting.md

```diff
@@ -232,14 +232,19 @@ One approach would be to use a factory function:
       imports: [ConfigModule],
       inject: [ConfigService],
       useFactory: (config: ConfigService) => ([{
-        ttl: config.get('THROTTLE_TTL'),
-        limit: config.get('THROTTLE_LIMIT'),
+        throttlers: [
```

Comment:
I don't think this example should be modified. 
The `throttlers` property of the options object if I am not mistaken is used on custom storage, and it always wraps `ttl` and `limit` in an array, perhaps a separate code snippet should be created for this case. Right @jmcdo29  ? 😄

Reply:
@Tony133 It was my mistake. You are right. :)

Reply:
@sinapirani, no problem 😉👍, at most edit this PR by adding a small section dedicated to the option: `throttlers` if you think it should be documented better or create a new PR, but maybe first ask @jmcdo29 how he wants to document this part 😉, I know he was working on it these weeks 🙂